app/components/filter-bar.js

```diff
@@ -1,20 +1,40 @@
-import Component from '@ember/component';
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
 import { debounce } from '@ember/runloop';
+import { tracked } from '@glimmer/tracking';
 
-export default Component.extend({
-  filter: null,
-  classNames: ['d-none d-md-flex'],
-  sortedAttribute: null,
-  sortableAttributes: null,
-  sortedAscending: null,
-  filterDebounce: computed('', {
-    set(key, value) {
-      debounce(this, this.setFilter, value, 250);
-      return null;
-    },
-  }),
-  setFilter(filter) {
-    this.set('filter', filter);
-  },
-});
+export default class FilterBar extends Component {
+  @tracked _immediateFilter;
+
+  get sortedAttribute() {
```

Comment:
Ohh these getters and setters are here to act as a kind of a proxy because of the way ember's built-in input components work (looks like they kind of still have a two way binding setup), I understand 👍🏿

Reply:
Yup. Though not all of them are two way (such as sortableAttributes, which is one way. So I should probably just use @sortableAttributes for that one